view/frontend/web/js/view/express-pay.js

```diff
@@ -0,0 +1,58 @@
+define([
+    'uiComponent',
+    'Bold_CheckoutPaymentBooster/js/model/express-pay',
+    'Magento_Customer/js/model/customer',
+    'ko'
+], function (
+    Component,
+    expressPay,
+    customer,
+    ko
+) {
+    'use strict';
+
+    /**
+     * PayPal Express pay button component.
+     */
+    return Component.extend({
+        defaults: {
+            template: 'Bold_CheckoutPaymentBooster/express-pay',
+            displayArea: '',
+            isVisible: true,
+        },
+        initialize: function () {
+            this._super();
+            
+            this.isVisibleObservable = ko.observable(this.isVisible);
```

Comment:
Is it necessary to have two variables here? Let me play with this and check...

Reply:
The `ko.observable` needs to be defined outside of the `defaults` section otherwise both of the blocks share that observable. That is what was causing the issue from yesterday where setting the visibility was applying to both. Let me know if you find a better way of handling it.

Reply:
Pushed another commit with the changes from our pairing session today. 

Details: we further revised the approach for injecting the buttons so only 1 new component is added in the xml. This allowed us to clean up the js to eliminate a lot of the conditional logic including how the ko observable is handled.